src/v0/destinations/marketo_bulk_upload/util.js

```diff
@@ -1,436 +0,0 @@
-  let fieldArr = [];
-  const fieldMap = {}; // map to store field name and data type
-  // ref: https://developers.marketo.com/rest-api/endpoint-reference/endpoint-index/#:~:text=Describe%20Lead2,leads/describe2.json
-  const { processedResponse: fieldSchemaMapping } = await handleHttpRequest(
-    'get',
-    `https://${munchkinId}.mktorest.com/rest/v1/leads/describe2.json`,
-    {
-      params: {
-        access_token: accessToken,
-      },
-    },
-    {
-      destType: 'marketo_bulk_upload',
-      feature: 'transformation',
-      endpointPath: '/leads/describe2.json',
-      requestMethod: 'GET',
-      module: 'router',
-    },
-  );
-  if (fieldSchemaMapping.response.errors) {
-    handleCommonErrorResponse(
-      fieldSchemaMapping,
-      'Error while fetching Marketo Field Schema',
-      'FieldSchemaMapping',
-    );
-  }
-  if (
-    fieldSchemaMapping.response?.success &&
-    fieldSchemaMapping.response?.result.length > 0 &&
-    fieldSchemaMapping.response?.result[0]
-  ) {
-    fieldArr =
-      fieldSchemaMapping.response.result && Array.isArray(fieldSchemaMapping.response.result)
-        ? fieldSchemaMapping.response.result[0]?.fields
-        : [];
-
-    fieldArr.forEach((field) => {
-      fieldMap[field?.name] = field?.dataType;
-    });
-  } else {
-    throw new RetryableError(
-      `Failed to fetch Marketo Field Schema due to error ${JSON.stringify(fieldSchemaMapping)}`,
-      500,
-      fieldSchemaMapping,
-    );
-  }
-  return fieldMap;
-};
-
-/**
- * Compares the data types of the fields in an event message with the expected data types defined in the field schema mapping.
- * Identifies any mismatched fields and returns them as a map of job IDs and the corresponding invalid fields.
- *
- * @param {object} event - An object containing an `input` array of events. Each event has a `message` object with field-value pairs and a `metadata` object with a `job_id` property.
- * @param {object} fieldSchemaMapping - An object containing the field schema mapping, which includes the expected data types for each field.
- * @returns {object} - An object containing the job IDs as keys and the corresponding invalid fields as values.
- */
-const checkEventStatusViaSchemaMatching = (event, fieldMap) => {
-  const mismatchedFields = {};
-  const events = event.input;
-  events.forEach((ev) => {
-    const { message, metadata } = ev;
-    // eslint-disable-next-line @typescript-eslint/naming-convention
-    const { job_id } = metadata;
-
-    Object.entries(message).forEach(([paramName, paramValue]) => {
-      const expectedDataType = SCHEMA_DATA_TYPE_MAP[fieldMap[paramName]];
-      const actualDataType = typeof paramValue;
-
-      if (
-        isDefinedAndNotNull(paramValue) &&
-        !mismatchedFields[job_id] &&
-        actualDataType !== expectedDataType
-      ) {
-        mismatchedFields[job_id] = `invalid ${paramName}`;
-      }
-    });
-  });
-  return mismatchedFields;
-};
-
-module.exports = {
```

Comment:
extend eslint config to find missing imports - if any of these utils are being referenced anywhere else

Reply:
Manually verfied no referecens are present, the eslint changes were not actually flagging need properly check later.